.github/workflows/coverity.yml

```diff
@@ -17,27 +17,22 @@ jobs:
     runs-on: ubuntu-22.04
     if: github.repository == 'systemd/systemd'
     env:
-      COVERITY_SCAN_BRANCH_PATTERN:     "${{ github.ref}}"
-      COVERITY_SCAN_NOTIFICATION_EMAIL: ""
+      COVERITY_SCAN_BRANCH_PATTERN:     "${{ github.ref }}"
       COVERITY_SCAN_PROJECT_NAME:       "${{ github.repository }}"
-      # Set in repo settings -> secrets -> repository secrets
+      COVERITY_SCAN_BUILD_COMMAND:      "ninja -C cov-build"
+      # Set in repo settings -> secrets -> actions
       COVERITY_SCAN_TOKEN:              "${{ secrets.COVERITY_SCAN_TOKEN }}"
-      CURRENT_REF:                      "${{ github.ref }}"
+      COVERITY_SCAN_NOTIFICATION_EMAIL: "${{ secrets.COVERITY_SCAN_NOTIFICATION_EMAIL }}"
+      # Auxiliary variables utilized by the coverity script
+      TRAVIS_BRANCH:                    "${{ github.ref }}"
```

Comment:
I'm not sure the script should keep carrying these variables. I'd just leave `COVERITY_SCAN_TOKEN` and `COVERITY_SCAN_NOTIFICATION_EMAIL` and drop everything else. More generally I'd just borrow https://github.com/dbus-fuzzer/dfuzzer/blob/master/.github/workflows/coverity.sh, replace `author_email` with `COVERITY_SCAN_NOTIFICATION_EMAIL` and also turn shellcheck on.

Reply:
That sounds like a better approach, thanks. I wanted to use the Coverity script as vanilla as possible to make future updates easier, but I completely forgot you did most of the legwork already in dfuzzer regarding the script cleanup, so I'll go ahead and use that instead.

Reply:
@mrc0mmand I think Travis is long gone (in terms of testing open source projects) so it's unlikely that that script will get updated updated very often (as far as I can tell it isn't even aware of the right status yet). I have been planning to replace it but I have to admit I was afraid to touch it here. Last time I tried something went wrong and I put it off :-) 

FWIW Coverity Scan will be updated on August 14 so this PR should probably be merged before or after that date to avoid weird side effects.

Reply:
> @mrc0mmand I think Travis is long gone (in terms of testing open source projects) so it's unlikely that that script will get updated updated very often (as far as I can tell it isn't even aware of the right status yet). I have been planning to replace it but I have to admit I was afraid to touch it here. Last time I tried something went wrong and I put it off :-)

Hehe, yeah. And the fact you can't easily test it/debug it doesn't help either :-)

> 
> FWIW Coverity Scan will be updated on August 14 so this PR should probably be merged before or after that date to avoid weird side effects.

That's definitely my intention (as the update information was why I started digging into this).